vulture_os/services/frontend/models.py

```diff
@@ -1477,6 +1478,13 @@ def rsyslog_only_conf(self):
         """ Check if this frontend has only rsyslog configuration, not haproxy at all """
         return self.mode == "impcap" or (self.mode == "log" and (self.listening_mode in ("udp", "file", "api", "kafka", "redis")))
 
+    @property
+    def has_tls(self):
+        for listener in self.listener_set.all():
+            if listener.is_tls:
+                return True
+        return False
+
```

Comment:
```suggestion
```
seems unused

Reply:
Yes it is right now, you're right. But the function is not useless, just unused.
We will use this function in the futur.